sound/pci/hda/patch_realtek.c

```diff
@@ -8034,6 +8035,22 @@ static const struct hda_fixup alc269_fixups[] = {
 	                 { }
 	         },
 	 },
+	 [ALC269VB_FIXUP_AYANEO_AIR_PLUS] = {
+	         .type = HDA_FIXUP_PINS,
+	         .v.pins = (const struct hda_pintbl[]) {
+	                 { 0x12, 0x90a60130 },
+	                 { 0x14, 0x90170110 },
+	                 { 0x17, 0x40000000 },
+	                 { 0x18, 0x04a19040 },
+	                 { 0x19, 0x411111f0 },
+	                 { 0x1a, 0x90170110 },
```

Comment:
We only need this pin to make the audio function. We could also use the pin cfg from the loki mini pro for 0x1a.
I have to test if 0x14 is doing something on the air plus 7320u (mendocino).

Reply:
It's quite possible that Pin 0x14 is not connected on the Air Plus just like the Ayn Loki Mini Pro. That said, I am all for code reuse if we can get away with it, what do you propose we call this fix instead of what I have doing which is VENDOR_MODEL.

Reply:
Yes, lets call the ayn Loki Mini pin cfg table.

Reply:
Keep in mind that not all Aya Neo Air Plus have the same pinouts. My unit didn't need a quirk at all.